services/web/server/src/simcore_service_webserver/projects/_crud_api_create.py

```diff
@@ -459,6 +466,17 @@ async def create_project(  # pylint: disable=too-many-arguments,too-many-branche
                 for gid, access in workspace.access_rights.items()
             }
 
+        _project_product_name = await _projects_repository.get_project_product(
+            project_uuid=new_project["uuid"]
+        )
+        assert (
+            _project_product_name == product_name  # nosec
+        ), "Project product name mismatch"
```

Comment:
you have the if below, unsure how useful this is, just let the below error propagate